install/build_from_source.md

```diff
@@ -56,7 +56,57 @@ To have ROOT setup automatically at each login, that command can be appended to
 
 ## ROOT, Python and PyROOT
 
-> TODO: explain what build options are relevant, what they do, Python2/Python3, etc.
+PyROOT, the set of Python bindings of ROOT, changed its structure and build/installation process in v6.22. In the following the main aspects are summarized for both after and before v6.22.
+
+### After v6.22
+
+The main feature introduced in v6.22 concerning the PyROOT build system is the possibility to build for both Python3 and Python2 (MultiPython), available only if the version of CMake used to build is >= 3.14. In the following, build and installation processes both with CMake >= 3.14 and < 3.14 are described. 
+
+#### CMake >= 3.14
+
+In this case, PyROOT libraries are built by default with both Python3 and Python2. For each Python version X.Y used to build PyROOT (e.g. 3.8, 2.7, etc.) the following libraries (containing the C++ extensions) will appear both in the build directory and in the installation directory:
+- `libROOTPythonizationsX_Y.so`
+- `libcppyX_Y.so`
+- `libcppyy_backendX_Y.so`
+
+The following pure Python modules will appear as well:
+- `ROOT`
+- `cppyy`
+- `cppyy_backend`
+- `JupyROOT`
+- `JsMVA`
+
+If no option is specified, PyROOT will be built for the most recent Python3 and Python2 versions that CMake can find. If only one version can be found, PyROOT will be built only for that version. Moreover, for a given Python installation to be considered, it must provide both the Python interpreter (binary) and the development package. To build PyROOT, it is thus suggested to verify that python-dev is present and [install it](https://pypi.org/project/python-dev-tools/) if not.
```

Comment:
Isn't `https://pypi.org/project/python-dev-tools/` a Python package that installs a set of tools for Python development? 

https://python-dev-tools.readthedocs.io/en/latest/readme.html#features

I think here you refer to the `pythonX-dev`/`pythonX-devel` package that provides the Python headers and libraries.

Reply:
yes, removed the hyperlink